nextflow_schema.json

```diff
@@ -630,5 +630,360 @@
         {
             "$ref": "#/definitions/institutional_config_options"
         }
-    ]
-}
+    ],
+    "properties": {
```

Comment:
If you really want to add all these parameters they should be added to the nfcore schema in the category:
"dia_spectral_library_search" and each of the parameters should have a description. I would probably hide these additional ones as well, since they are quite specialized ("hidden": "true")